commons/src/main/kotlin/com/simplemobiletools/commons/dialogs/PermissionRequiredDialog.kt

```diff
@@ -0,0 +1,42 @@
+package com.simplemobiletools.commons.dialogs
+
+import android.app.Activity
+import androidx.appcompat.app.AlertDialog
+import com.simplemobiletools.commons.R
+import com.simplemobiletools.commons.extensions.getAlertDialogBuilder
+import com.simplemobiletools.commons.extensions.setupDialogStuff
+import kotlinx.android.synthetic.main.dialog_message.view.message
+
+class PermissionRequiredDialog(
+    activity: Activity,
+    message: String = "",
+    messageId: Int = R.string.no_permission,
+    val cancelOnTouchOutside: Boolean = true,
+    dialogTitle: String = "",
+    dialogTitleId: Int = R.string.permission_required,
+    val callback: () -> Unit,
+) {
+    private var dialog: AlertDialog? = null
+    init {
+        val title: String by lazy { dialogTitle.ifEmpty { activity.getString(dialogTitleId) } }
```

Comment:
ah forgot about this before accepting the PR, I mean do not use lazy. Lets do a casual if () check